[Bug]: build libjxl failed ndk25

### Problem description

-- Build files have been written to: /home/builder/.termux-build/libjxl/build
[1/208] Generating cjxl.1
FAILED: cjxl.1 /home/builder/.termux-build/libjxl/build/cjxl.1
cd /home/builder/.termux-build/libjxl/build && /usr/bin/python3.10 /usr/bin/a2x --format manpage --destination-dir="/home/builder/.termux-build/libjxl/build" /home/builder/.termux-build/libjxl/src/doc/man/cjxl.txt
  File "/usr/bin/a2x", line 2
    python3 -m asciidoc.a2x "$@"
               ^^^^^^^^
SyntaxError: invalid syntax
[2/208] Generating djxl.1
FAILED: djxl.1 /home/builder/.termux-build/libjxl/build/djxl.1
cd /home/builder/.termux-build/libjxl/build && /usr/bin/python3.10 /usr/bin/a2x --format manpage --destination-dir="/home/builder/.termux-build/libjxl/build" /home/builder/.termux-build/libjxl/src/doc/man/djxl.txt
  File "/usr/bin/a2x", line 2
    python3 -m asciidoc.a2x "$@"
               ^^^^^^^^
SyntaxError: invalid syntax
[12/208] Building CXX object third_party/CMakeFiles/lodepng.dir/lodepng/lodepng.cpp.o
ninja: build stopped: subcommand failed.
builder@3734a7c6abeb:~/termux-packages$

[libjxl.txt](https://github.com/termux/termux-packages/files/9333825/libjxl.txt)


### What steps will reproduce the bug?

build

### What is the expected behavior?

_No response_

### System information

termux-info:

```

```

--- a/packages/libjxl/a2x-is-shell-script.patch
+++ b/packages/libjxl/a2x-is-shell-script.patch
@@ -1,0 +1,13 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -355,8 +355,8 @@ if (ASCIIDOC_PY_FOUND)
+     # does not recognize it.
+     add_custom_command(
+       OUTPUT "${PAGE}.1"
+-      COMMAND "${ASCIIDOC_PY}"
+-      ARGS "${ASCIIDOC}"
++      COMMAND "${ASCIIDOC}"
++      ARGS
+         --format manpage --destination-dir="${CMAKE_CURRENT_BINARY_DIR}"
+         "${CMAKE_CURRENT_SOURCE_DIR}/doc/man/${PAGE}.txt"
+       MAIN_DEPENDENCY "${CMAKE_CURRENT_SOURCE_DIR}/doc/man/${PAGE}.txt")

